### PR TITLE
8301942: java/net/httpclient/DigestEchoClientSSL.java fail with -Xcomp

### PR DESCRIPTION
**Pull request by sunny868**
This test failed with VM_OPTIONS=-Xcomp and CONF=fastdebug on the LOONGARCH64 architecture.

java.lang.AssertionError: WARNING: tracker for HttpClientImpl(1) has outstanding operations:
    Pending HTTP Requests: 0
    Pending HTTP/1.1 operations: 0
    Pending HTTP/2 streams: 0
    Pending WebSocket operations: 0
    Pending TCP connections: 1
    Total pending operations: 0
    Facade referenced: false
    Selector alive: true
Found 0 client still active, with 0 operations still pending out of 1 tracked clients.

This PR fix the issue, Please help review it.

Thanks.

---------
### Progress
- [x] Change must be properly reviewed (1 review required, with at least 1 [Reviewer](https://openjdk.org/bylaws#reviewer))
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue

### Issue
 * [JDK-8301942](https://bugs.openjdk.org/browse/JDK-8301942): java/net/httpclient/DigestEchoClientSSL.java fail with -Xcomp

### Reviewers
 * [Daniel Fuchs](https://openjdk.org/census#dfuchs) (@dfuch - **Reviewer**)

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.org/jdk pull/12449/head:pull/12449` \
`$ git checkout pull/12449`

Update a local copy of the PR: \
`$ git checkout pull/12449` \
`$ git pull https://git.openjdk.org/jdk pull/12449/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 12449`

View PR using the GUI difftool: \
`$ git pr show -t 12449`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.org/jdk/pull/12449.diff">https://git.openjdk.org/jdk/pull/12449.diff</a>

</details>

### CODE REVIEW

**Review by dfuch — APPROVED** [review]
Sounds reasonable to me

**Comment by dfuch** [comment]
I will sponsor this change. Let me test it locally first.

**Comment by RogerRiggs** [comment]
@sunny868 Can you summarize what is different about the loongaarch platform and port? Clock speeds, memory organization, memory sizes, etc?
I'm cautious about making multiple changes related -Xcomp tests without knowing whether the problem might indicate something closer to the implementation.

**Comment by sunny868** [comment]
> @sunny868 Can you summarize what is different about the loongaarch platform and port? Clock speeds, memory organization, memory sizes, etc? 

The basic information of the machine is as follows:
<pre><code class="shell">
Architecture:        loongarch64
Byte Order:          Little Endian
CPU(s):              4
On-line CPU(s) list: 0-3
Thread(s) per core:  1
Core(s) per socket:  4
Socket(s):           1
NUMA node(s):        1
CPU family:          Loongson-64bit
Model name:          Loongson-3A5000-HV
BogoMIPS:            5000.00
L1d cache:           64K
L1i cache:           64K
L2 cache:            256K
L3 cache:            16384K
NUMA node0 CPU(s):   0-3

CPU MHz			: 2500.00

Mem:           16G
</code></pre>
Since LoongArch is still a relatively new architecture, individual instructions are inefficient and not as abundant in instructions, it is lower than aarch64 in terms of compilation performance (compiling a method will take more time) and running performance, so many problems with `-Xcomp` are exposed.

**Comment by RogerRiggs** [comment]
Thanks for the background; looks ok to me.
